cypress/integration/FarmerProfile-tests.js

```diff
@@ -0,0 +1,125 @@
+describe('farmer profile page', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://wheat-cute-api.herokuapp.com/graphql', (req) => {
+      req.reply({
+        statusCode: 200,
+        fixture: 'allFarmersProfile.json'
+      })
+    })
+
+    // cy.visit('https://wheat-cute.herokuapp.com/farms/2')
```

Comment:
Good thinking putting this in here now, so that we can simply switch it over once we deploy.